package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/checkout",
-  "version": "1.286.1",
+  "version": "1.286.2-prerelease.1674725660681",
```

Comment:
I think this is unintentional?

Reply:
Hmm... I thought I removed that. Thanks Tom

Reply:
Give me a sec

Reply:
Sorry about that 🙏

Reply:
LGTM! 👍